lib/bridge/IrcHandler.js

```diff
@@ -21,6 +21,8 @@ const MODES_TO_WATCH = [
     "s"
 ];
 
+const NICK_USERID_CACHE_MAX = 512;
```

Comment:
not worth blocking this PR on it, but it would be interesting to have metrics to see how often this cache is overrun. 512 channels for freenode doesn't sound like a lot, but might be given most of them are probably quiet.

Reply:
Given the cost of generating this list is small (relatively), I can live with it. Remember we are dropping things from this cache whenever someone joins or leaves so I'd expect it to stay fairly low.

Reply:
Yea, more of a thing where I love to see graphs for things. I don't suspect it'll actually be a problem.